package.json

```diff
@@ -149,7 +149,7 @@
     "@types/js-cookie": "^3.0.2",
     "@types/lodash.debounce": "4.0.7",
     "@types/new-relic-browser": "0.1118.0",
-    "@types/node": "14.14.10",
+    "@types/node": "^16.11.47",
```

Comment:
It looks like we actually require >= 16.13.x
https://github.com/evergreen-ci/spruce/blob/eac039ab65712d6160aad369ce2dab17b5ff3526/package.json#L234

Reply:
it doesn't exist :/ 
https://releasealert.dev/npm/@types/node
16.11.47 is the latest release for node version 16.